### PR TITLE
Fix /lwc admin purge to let player names work again 

### PR DESCRIPTION
**Pull request by trainphreak**
Replacement PR for #1

### CODE REVIEW

**Comment by trainphreak** [comment]
Fast hopper protection is a workaround for the lag-tastic result of large numbers of active hoppers on an LWC-enabled server. The massive number of InventoryMoveItemEvents create a horde of database lookups that bottlenecks the main thread, even if none of the hoppers are protected (because the lookups are to look for applicable protections).

File: core/src/main/resources/plugin.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name: LWC
 main: com.griefcraft.lwc.LWCPlugin
-version: ${project.version}
+version: 4.5.1-SNAPSHOT
```
Comment (dmulloy2) [inline core/src/main/resources/plugin.yml]: Maven automatically fills the version info during compilation

Comment (dmulloy2) [inline core/src/main/resources/plugin.yml]: Gotcha. Did /version LWC have the same issue?

Comment (trainphreak) [inline core/src/main/resources/plugin.yml]: Yep, and they both display the website as well, even with your latest changes pulled in that remove it from plugin.yml.

Comment (dmulloy2) [inline core/src/main/resources/plugin.yml]: It appears to work fine when I compile it on my Jenkins: http://ci.dmulloy2.net/job/LWC/

**Comment by trainphreak** [comment]
Oh fun, I get to dig into Maven JAR shading then.

**Pull request merged**